### PR TITLE
[Django 3.0] ImportError: cannot import name 'python_2_unicode_co…

### PR DESCRIPTION
**Pull request by agl29**
…mpatible' from 'django.utils.encoding'

## What changes were proposed in this pull request?

  from django.utils.encoding import python_2_unicode_compatible
 to
   from six import python_2_unicode_compatible

### CODE REVIEW

File: desktop/core/src/desktop/lib/django_forms.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 from django.forms.widgets import MultiWidget, Select, TextInput, Textarea, HiddenInput, Input
 from django.utils import formats
 from django.utils.safestring import mark_safe
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```
Comment (romainr) [inline desktop/core/src/desktop/lib/django_forms.py:36]: Not used actually? (re-imported below?)

File: desktop/core/src/desktop/lib/django_forms.py [hunk 2]
```diff
@@ -51,7 +51,7 @@
 try:
   from django.utils.encoding import StrAndUnicode
 except ImportError:
-  from django.utils.encoding import python_2_unicode_compatible
+  from six import python_2_unicode_compatible
```
Comment (romainr) [inline desktop/core/src/desktop/lib/django_forms.py:54]: For the record this is a no-op in Python 3
https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.encoding.python_2_unicode_compatible

**Review by romainr — APPROVED** [review]
nit: [Django Upgrade] --> [django 3.0]

nit: please include more description/context on the deprecation in the commit message (like I put below..), it helps understand if we are going to break things

File: desktop/core/src/desktop/lib/django_forms.py (hunk 1 above)
Comment (agl29) [reply desktop/core/src/desktop/lib/django_forms.py]: done

**Comment by agl29** [comment]
> nit: [Django Upgrade] --> [django 3.0]
> 
> nit: please include more description/context on the deprecation in the commit message (like I put below..), it helps understand if we are going to break things

done, okay

**Pull request merged**